### PR TITLE
adds GDK Xbox One and Scarlett configurations

### PR DESCRIPTION
**Pull request by cigumo**
Requires GENie PR: https://github.com/bkaradzic/GENie/pull/546

### CODE REVIEW

File: scripts/toolchain.lua [hunk 1]
```diff
@@ -959,6 +965,32 @@ function toolchain(_buildDir, _libDir)
...
+		defines {
+			"_WIN64",
+			"NOMINMAX",
+			"UNICODE",
+			"_UNICODE",
+		}
+		buildoptions {
+		    "/favor:AMD64"
+		}
+
```
Comment (cigumo) [inline scripts/toolchain.lua:980]: Forgot to add:  removeflags { "StaticRuntime" }

File: scripts/toolchain.lua [hunk 2]
```diff
@@ -959,6 +965,32 @@ function toolchain(_buildDir, _libDir)
...
```
Comment (cigumo) [inline scripts/toolchain.lua:993]: Forgot to add:  removeflags { "StaticRuntime" }

Comment (bkaradzic) [reply scripts/toolchain.lua:993]: Add it, I didn't merge PR yet.

File: scripts/toolchain.lua (hunk 1 above)
Comment (bkaradzic) [reply scripts/toolchain.lua:980]: Add it, I didn't merge PR yet.

Comment (cigumo) [reply scripts/toolchain.lua:980]: OK! Will resend a new PR.

**Comment by cigumo** [comment]
Closing to replace with cleaned up version.

**Pull request closed without merging**